multi-node/aws/.gitignore

```diff
@@ -1,4 +1,5 @@
 /artifacts/template.json
 /bin
-/clusters
-/cluster.yaml
+/assets
+/*.yaml
+*~
```

Comment:
"All files in the tree that end in tilda"
On Jan 25, 2016 6:43 PM, "Aaron Levy" <notifications@github.com> wrote:

In multi-node/aws/.gitignore
<https://github.com/coreos/coreos-kubernetes/pull/240#discussion_r50790070>:

> @@ -1,4 +1,5 @@
>  /artifacts/template.json
>  /bin
> -/clusters
> -/cluster.yaml
> +/assets
> +/*.yaml
> +*~

What specifically are you trying to ignore here? Can we have explicit files
instead of * expansion?

—
Reply to this email directly or view it on GitHub
<https://github.com/coreos/coreos-kubernetes/pull/240/files#r50790070>.